TLSharp.Tests.VS/TLSharpTestsVs.cs

```diff
@@ -0,0 +1,71 @@
+﻿
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TLSharp.Tests
+{
+    [TestClass]
+    public class TLSharpTestsVS : TLSharpTests
```

Comment:
I want to make TLSharpTests and TLSharpTestsNUnit classes. TlSharpTestsNUnit will be derived from mstest class. So you'll no difference between old tests and new tests. Can we do like this?